e2e/cypress/integration/enterprise/guest_accounts/member_invitation_ui_spec.js

```diff
@@ -105,4 +181,33 @@ describe('Guest Account - Member Invitation Flow', () => {
         cy.get('#confirmModalButton').should('be.visible').and('have.text', 'Yes, Discard').click();
         cy.get('.InvitationModal').should('not.exist');
     });
+
+    it('MM-18040 Verify Invite New/Existing Users', () => {
+        // # Login as new user and get the user id
+        cy.createNewUser().then((newUser) => {
+            cy.apiAddUserToTeam(testTeam.id, newUser.id);
+            cy.apiLogin(newUser.username, newUser.password);
+            cy.visit('/');
+            cy.visit(`/${testTeam.name}`);
+        });
+
+        // # Search and add an existing member by username who is part of the team
+        invitePeople(sysadmin.username, 1, sysadmin.username);
+
+        // * Verify the content and message in next screen
```

Comment:
This generic message might be fine as comment instead of indicating whether it's error/success, but on the PR description (where I supposed will be included in our release testing spreadsheet) which reads as:
> Search and add an existing member by username, who is part of the team and click Invite Members and verify the message

Specifically "verify the message", I suggest to make it explicit on what's expected.

Reply:
Sure. Makes sense. I will update the PR Description accordingly.